TUTORIAL.md

```diff
@@ -765,17 +782,19 @@ When I re-deployed with this change, I received a 100. *Huzzah!*
 
 You can find the source code associated with this article [on GitHub](https://github.com/oktadeveloper/okta-spring-boot-angular-pwa-example). If you find any bugs, please file an issue, or leave a comment on this post. Of course, you can always [ping me on Twitter](https://twitter.com/mraible) too.
 
-This article showed you how to develop a Spring Boot backend, and lock it down with Okta. You learned how to develop an Angular front end and use OpenID Connect to get an access token and securely communicate with the backend. You also learned how you can use the Stormpath Angular SDK to do the same thing. Finally, you saw how to deploy everything to Cloud Foundry and get a Lighthouse PWA score of 100.
+This article showed you how to develop a Spring Boot backend, and lock it down with Okta. You learned how to develop an Angular frontend and use OpenID Connect to get an access token and securely communicate with the backend. You also learned how you can use the Stormpath Angular SDK to do the same thing. Finally, you saw how to deploy everything to Cloud Foundry and get a Lighthouse PWA score of 100.
 
-To learn more about PWAs, check out some recent tutorials I wrote.
+To learn more about PWAs, check out some recent tutorials I wrote:
 
 * [Build Your First Progressive Web Application with Angular and Spring Boot](http://developer.okta.com/blog/2017/05/09/progressive-web-applications-with-angular-and-spring-boot) 
 * [Tutorial: Develop a Mobile App With Ionic and Spring Boot](http://developer.okta.com/blog/2017/05/17/develop-a-mobile-app-with-ionic-and-spring-boot)
 
-There's also a number of excellent resources by Google, Pluralsight, and Smashing Magazine.
+There's also a number of excellent resources by Google, Pluralsight, and Smashing Magazine:
 
 * Addy Osmani at Google I/O '17: [Production Progressive Web Apps With JavaScript Frameworks](https://youtu.be/aCMbSyngXB4)
 * Google's [Progressive Web Apps](https://developers.google.com/web/progressive-web-apps/) homepage, [step-by-step code lab](https://codelabs.developers.google.com/codelabs/your-first-pwapp/), and [instructor-led PWA training](https://developers.google.com/web/ilt/pwa/).
 * Pluralsight's [Getting Started with Progressive Web Apps](https://www.pluralsight.com/courses/web-apps-progressive-getting-started)
 * [A Beginner's Guide To Progressive Web Apps](https://www.smashingmagazine.com/2016/08/a-beginners-guide-to-progressive-web-apps/)
+
+// let's kill the last link here, and add your scotch post's url to the list above. i'll work on getting that URL for you
```

Comment:
Is that a circular dependency? I wrote this post for us to link to from scotch.io, and then we link back. I like it!